src/catkin_pkg/package_templates.py

```diff
@@ -294,10 +294,12 @@ def _create_targetlib_args(package_template):
 
 
 def _create_include_macro(package_template):
-    result = '# include_directories(include)'
+    result = '#  include'
     includes = []
     if package_template.catkin_deps:
         includes.append('${catkin_INCLUDE_DIRS}')
+    else:
+        includes.append('#  ${catkin_INCLUDE_DIRS}')
```

Comment:
For sure. I should have mentioned, not only am I a complete ROS noob, but my `python` consists of 2 fully and 1 partially completed coursera class... I suppose free, but not-super-foo help, is still free though :)